scripts/ci_script.sh

```diff
@@ -198,8 +198,8 @@ if [[ $GROUP == usage ]]; then
     jupyter labextension enable @jupyterlab/mock-extension --debug
     jupyter labextension uninstall @jupyterlab/mock-extension --debug
     jupyter labextension list 1>labextensions 2>&1
-    # bail if mock-extension was listed
-    cat labextensions | grep -q "mock-extension" && exit 1
+    # check the federated extension is still listed after jupyter labextension uninstall
+    cat labextensions | grep -q "mock-extension"
```

Comment:
Switched back to using `grep` to check for the extension, since `jupyter labextension uninstall @jupyterlab/mock-extension --debug` returns exist code 0 also if it refuses to uninstall the extension.